src/components/modals/Dialog.jsx

```diff
@@ -71,7 +71,14 @@ const Dialog = (props) => {
                     tabIndex="-1"
                     aria-modal="true"
                     onKeyDown={handleKeyDown}
-                    style={{ maxWidth: `${props.width}px` }}
+                    style={{
+                      maxWidth: `${props.width}${
+                        typeof props.width === 'number' ? 'px' : ''
+                      } `,
+                      maxHeight: `${props.height}${
+                        typeof props.height === 'number' ? 'px' : ''
+                      }`,
+                    }}
```

Comment:
I'm thinking we shouldn't accept numbers in the future, to allow developers to supply their own units. That being said, I added this code to detect numbers and add pixels, and if it's not a number, to not add anything, so that we wouldn't unnecessarily introduce a breaking change here.

Reply:
This is nice.
If we begin accept strings, we should also update the proptypes and their jsdoc comments

Reply:
I updated the prop type to say that it accepts strings, with the assumption that numbers are still supported, but it's sort of a soft deprecation to not list it, and then we'll rip it out in the next major.